cloud-functions/utils/requirements.txt

```diff
@@ -0,0 +1,55 @@
+antlr4-python3-runtime==4.8
+azure-core==1.25.0
+azure-identity==1.10.0
+azure-storage-blob==12.13.1
+cachetools==5.2.0
+certifi==2022.6.15
+cffi==1.15.1
+charset-normalizer==2.1.0
+click==8.1.3
+cloudevents==1.5.0
+coverage==6.4.4
+cryptography==37.0.4
+deprecation==2.1.0
+fhirpathpy==0.1.0
+Flask==2.2.2
+functions-framework==3.2.0
+google-auth==2.10.0
+gunicorn==20.1.0
+hl7==0.4.5
+idna==3.3
+isodate==0.6.1
+itsdangerous==2.1.2
+Jinja2==3.1.2
+MarkupSafe==2.1.1
+msal==1.18.0
+msal-extensions==1.0.0
+msrest==0.7.1
+numpy==1.23.2
+oauthlib==3.2.0
+packaging==21.3
+pandas==1.4.3
+phdi @ git+https://github.com/CDCgov/phdi@ebdd43120eec138373bc36900cf6ea7e25aedc29
```

Comment:
Should this target `main` rather than a specific commit?

Reply:
Eventually yes, but for now we are pointing to a specific version of the SDK project.  This isn't even necessary for the utils anyway.   I'll remove it for this requirements doc for now.